ballerina-tests/tests/http2_mutual_ssl_test.bal

```diff
@@ -97,8 +97,63 @@ http:ClientConfiguration http2MutualSslClientConf = {
 };
 
 @test:Config {}
-public function testHttp2MutualSsl() {
-    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf);
+public function testHttp2MutualSsl1() {
```

Comment:
Shall we update the test function signature as `public function testHttp2MutualSsl1() returns error?` and use `check` instead of `checkpanic` ? 

Check for other places as well.

Reply:
The issue is created to track this. https://github.com/ballerina-platform/ballerina-standard-library/issues/1718